pallets/fragments/src/lib.rs

```diff
@@ -1399,7 +1405,7 @@ pub mod pallet {
 			}
 
 			let detach_hash = DetachHash::Instance(definition_hash, Compact(edition_id), Compact(copy_id));
-			let detach_request = DetachRequest { hash: detach_hash.clone(), target_chain, target_account};
+			let detach_request = DetachRequest { hash: detach_hash.clone(), target_chain, target_account: target_account.into() };
```

Comment:
I suppose @alebaffa you do something similar iirc anyway already. @aang114 Alessandro iirc also ensures non zero! Which is a good touch in some cases.

Reply:
I ensure non zero in pallet-oracle, [here](https://github.com/fragcolor-xyz/clamor/blob/458a5f20d20d805e768dbfcd15ea5b4d9c0041a2/pallets/oracle/src/lib.rs#L424):
```rust
ensure!(round_id.gt(&U256::zero()), "Price from oracle is 0");
ensure!(price.gt(&U256::zero()), "Price from oracle is >= 0");
```

Reply:
err we are talking of bounded vectors though